### PR TITLE
Favicon MV3 example

### PR DESCRIPTION
**Pull request by solomonkinard**

### CODE REVIEW

**Review by dotproto — CHANGES_REQUESTED** [review]
One suggestion and a couple questions.

File: api/favicon/content.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<script src="content.js"></script><body></body>
```
[the remark this answers is not in the document]
Comment (solomonkinard) [reply api/favicon/content.html]: Done.

File: api/favicon/content.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+window.onload = e => {
+  const img = document.createElement('img');
+  const url = `chrome-extension://${chrome.runtime.id}/_favicon/?page_url=`;
```
[the remark this answers is not in the document]
Comment (solomonkinard) [reply api/favicon/content.js]: Done.

**Comment by solomonkinard** [comment]
Thanks.

**Review by dotproto — APPROVED** [review]
Thanks, @solomonkinard!

**Pull request merged**